CHANGELOG.rst

```diff
@@ -4,6 +4,12 @@ Changelog
 in development
 --------------
 
+Added
```

Comment:
I think the PR belongs to `Changed` category as we change the default logic here and it's not a brand new feature or functionality.